### ISSUE TITLE
ML: Pretrained Model #1 (MNIST): Configuration#7: Training model and store the results in the directory

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#262]
1. Put the training on the Grid by using jetscape-ml-tensorflow-nn-mnist.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
Epoch 00050: val_accuracy improved from 0.82709 to 0.83110, saving model to /wsu/home/gy/gy40/gy4065/hm.jetscapeml.data/simulation_results/simulation-results-nn-mnist-1200K-config-07/models_MATTER_vs_LBT_Hadron_ch1/hm_jetscape_ml_model_best.h5

840000/840000 [==============================] - 14s 17us/sample - loss: 0.5635 - accuracy: 0.8145 - val_loss: 0.5604 - val_accuracy: 0.8311

**Comment by hmehryar** [comment]
port into Excel for overleaf
